proto/serverless/instrumentation/tags/v1/tags.proto

```diff
@@ -0,0 +1,306 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 100  //
+  // ========================================== //
+
+  // ============================================ //
+  // Top level tags will have their underscores replaced //
+  // with periods when flattened. So for `aws_lambda` it will //
+  // be converted to `aws.lambda`//
+
+  // The root AWS Lambda Span tags.
+  optional AwsLambdaTags aws_lambda = 100;
+
+  // The AWS Lambda Initialization tags.
+  optional AwsLambdaInitializationTags aws_lambda_initialization = 101;
+
+  // The AWS Lambda Invocation tags.
+  optional AwsLambdaInvocationTags aws_lambda_invocation = 102;
+
+  // The AWS SDK Tags.
+  optional AwsSdkDynamodbTags aws_sdk_dynamodb = 103;
+  optional AwsSdkSqsTags aws_sdk_sqs = 104;
+  optional AwsSdkSnsTags aws_sdk_sns = 105;
+
+
+  // The HTTP Tags.
+  optional HttpTags http = 106;
+
+  // The HTTPS Tags.
+  optional HttpTags https = 107;
+
+  // The Express Tags.
+  optional ExpressTags express = 108;
+
+  // The AWS Trace Context Tags.
+  optional AwsTags aws = 109;
+}
+
+message SlsSdkTags {
+  // A Serverless Platform OrgId.
+  string org_id = 1;
+  // The platform that was instrumented. Currently Lambda is the only supported platform.
+  string platform = 2;
+  // The service that was instrumented. For Lambda this will be the function name by default.
+  string service = 3;
+  // The region that instrumentation was performed in. This is used to determine which Serverless Ingest API to use.
+  string region = 4;
```

Comment:
As I addressed elsewhere, 

The only things within spans that can be sourced from cloudwatch will be,

1. Duration
2. LogGroup and LogStream

Everything else I'm not seeing how we could get.

> We pave the path for the support of runtimes for which no SDK/internal extension exists

In no way will we be supporting any sort of span/traces without an extension/SDK so not sure why this concern matters. If an extension/SDK is not used then all users will get is logs and metrics.

Reply:
> In no way will we be supporting any sort of span/traces without an extension/SDK

Why exactly? When and where it was decided?

I remember that when we working with external + internal extension. There was a clear agreement, that external extension should in the end support runtime cases for which we do not have SDK/internal extension

Reply:
But what are we supporting if there isn't an SDK? If you want spans/traces you need something to create them. So whether we create a python SDK ourselves or a member of the community does, they will need to adhere to the expectation of what values are attached.

Reply:
> If you want spans/traces you need something to create them.

Spans/traces can be created from logs, and we do not need to inject SDK to have the basic ones (as `aws.lambda`, `aws.lambda.invocation`, `aws. lambda.initialisation).

I wouldn't attach to the idea, that no internal extension means no console instrumentation, as I believe we never wanted to imply such restriction.

Reply:
That is fine if we potentially want to do that at some point. But it is not something we are supporting initially. I also don't love the idea of having incomplete spans/traces from logs, we will need to think of this separately internally as a feature once this initial release is complete.

Reply:
>  I also don't love the idea of having incomplete spans/traces from logs

It's impossible to have them complete and we already agreed that spans, as generated by the extension, will be completed after we read data from CW.

And my point is, that best if that's the case with all the properties that can be read from CW context.

This will help to reduce the payload, and it's a step toward extensionless integrations, which I believe will definitely at some point be pursued.

Reply:
Given what we discussed I think this is fine, assuming that we only take from cloudwatch,

1. Duration
2. Max Memory
3. LogGroup/Logstream
4. Error Outcome

Reply:
Can we add function name and  _constant_ properites as `platform`?

Ideally if we comment at each property, how it's sourced, so there's no confusion around that.